platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/MapboxApplication.java

```diff
@@ -70,7 +73,10 @@ private void initializeMapbox() {
     String accessToken = TokenUtils.getMapboxAccessToken(getApplicationContext());
     validateAccessToken(accessToken);
     Mapbox.getInstance(getApplicationContext(), accessToken);
-    Telemetry.updateDebugLoggingEnabled(true);
+    TelemetryDefinition telemetry = Mapbox.getTelemetry();
+    if (telemetry != null) {
```

Comment:
What do you think about removing the null-check here, or crashing the app with a clearer exception, to catch start-up telemetry regressions during internal testing?

Reply:
added:
```java
    if (telemetry == null) {
      throw new IllegalStateException("Telemetry was unavailable during test application start.");
    }
```